### ISSUE TITLE
Error installing v1.0.0

### ISSUE THREAD

**Issue by cristina-defran** [sysrepo/sysrepo-python#29]
Hello,

I tried to install the latest v1.0.0 release for compatibility with 2.x versions but I get the following error:

```
 > [build 11/11] RUN pip3 install sysrepo:                                                                                
#0 3.738 Collecting sysrepo                                                                                               
#0 3.858   Downloading sysrepo-1.0.0.tar.gz (32 kB)                                                                       
#0 3.931   Installing build dependencies: started                                                                         
#0 14.20   Installing build dependencies: finished with status 'done'                                                     
#0 14.20   Getting requirements to build wheel: started
#0 14.99   Getting requirements to build wheel: finished with status 'done'
#0 15.00   Installing backend dependencies: started
#0 18.98   Installing backend dependencies: finished with status 'done'
#0 18.99     Preparing wheel metadata: started
#0 26.18     Preparing wheel metadata: finished with status 'done'
#0 26.22 Requirement already satisfied: libyang>=2.1 in /usr/local/lib/python3.8/dist-packages (from sysrepo) (2.1.0)
#0 26.24 Requirement already satisfied: cffi; platform_python_implementation != "PyPy" in /usr/lib/python3/dist-packages (from sysrepo) (1.14.0)
#0 26.24 Building wheels for collected packages: sysrepo
#0 26.25   Building wheel for sysrepo (PEP 517): started
#0 33.27   Building wheel for sysrepo (PEP 517): finished with status 'error'
#0 33.27   ERROR: Command errored out with exit status 1:
#0 33.27    command: /usr/bin/python3 /tmp/tmpjxnvppfw build_wheel /tmp/tmpf1pzpwjj
#0 33.27        cwd: /tmp/pip-install-_ul0o5dc/sysrepo
#0 33.27   Complete output (38 lines):
#0 33.27   running bdist_wheel
#0 33.27   running build
#0 33.27   running build_py
#0 33.27   creating build
#0 33.27   creating build/lib.linux-x86_64-cpython-38
#0 33.27   creating build/lib.linux-x86_64-cpython-38/sysrepo
#0 33.27   copying sysrepo/session.py -> build/lib.linux-x86_64-cpython-38/sysrepo
#0 33.27   copying sysrepo/errors.py -> build/lib.linux-x86_64-cpython-38/sysrepo
#0 33.27   copying sysrepo/subscription.py -> build/lib.linux-x86_64-cpython-38/sysrepo
#0 33.27   copying sysrepo/__init__.py -> build/lib.linux-x86_64-cpython-38/sysrepo
#0 33.27   copying sysrepo/util.py -> build/lib.linux-x86_64-cpython-38/sysrepo
#0 33.27   copying sysrepo/value.py -> build/lib.linux-x86_64-cpython-38/sysrepo
#0 33.27   copying sysrepo/connection.py -> build/lib.linux-x86_64-cpython-38/sysrepo
#0 33.27   copying sysrepo/change.py -> build/lib.linux-x86_64-cpython-38/sysrepo
#0 33.27   running egg_info
#0 33.27   writing sysrepo.egg-info/PKG-INFO
#0 33.27   writing dependency_links to sysrepo.egg-info/dependency_links.txt
#0 33.27   writing requirements to sysrepo.egg-info/requires.txt
#0 33.27   writing top-level names to sysrepo.egg-info/top_level.txt
#0 33.27   reading manifest file 'sysrepo.egg-info/SOURCES.txt'
#0 33.27   reading manifest template 'MANIFEST.in'
#0 33.27   adding license file 'LICENSE'
#0 33.27   writing manifest file 'sysrepo.egg-info/SOURCES.txt'
#0 33.27   copying sysrepo/py.typed -> build/lib.linux-x86_64-cpython-38/sysrepo
#0 33.27   running build_ext
#0 33.27   generating cffi module 'build/temp.linux-x86_64-cpython-38/_sysrepo.c'
#0 33.27   creating build/temp.linux-x86_64-cpython-38
#0 33.27   building '_sysrepo' extension
#0 33.27   creating build/temp.linux-x86_64-cpython-38/build
#0 33.27   creating build/temp.linux-x86_64-cpython-38/build/temp.linux-x86_64-cpython-38
#0 33.27   x86_64-linux-gnu-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -I/usr/include/python3.8 -c build/temp.linux-x86_64-cpython-38/_sysrepo.c -o build/temp.linux-x86_64-cpython-38/build/temp.linux-x86_64-cpython-38/_sysrepo.o -Werror -std=c99
#0 33.27   build/temp.linux-x86_64-cpython-38/_sysrepo.c: In function ‘_cffi_const_SR_CONN_CTX_SET_PRIV_PARSED’:
#0 33.27   build/temp.linux-x86_64-cpython-38/_sysrepo.c:976:12: error: ‘SR_CONN_CTX_SET_PRIV_PARSED’ undeclared (first use in this function); did you mean ‘LY_CTX_SET_PRIV_PARSED’?
#0 33.27     976 |   int n = (SR_CONN_CTX_SET_PRIV_PARSED) <= 0;
#0 33.27         |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~
#0 33.27         |            LY_CTX_SET_PRIV_PARSED
#0 33.27   build/temp.linux-x86_64-cpython-38/_sysrepo.c:976:12: note: each undeclared identifier is reported only once for each function it appears in
#0 33.27   error: command '/usr/bin/x86_64-linux-gnu-gcc' failed with exit code 1
#0 33.27   ----------------------------------------
#0 33.27   ERROR: Failed building wheel for sysrepo
#0 33.27 Failed to build sysrepo
#0 33.35 ERROR: Could not build wheels for sysrepo which use PEP 517 and cannot be installed directly
------
```

I am using **sysrepo v2.1.64**, **libyang v2.0.194**, **libnetconf2 v2.1.11** and **netopeer2-server v2.1.23**.

I saw [someone commenting the same error on a closed issue](https://github.com/sysrepo/sysrepo-python/issues/12#issuecomment-1136106651) 13 days ago, but seeing nobody answered, I thought it'd be better to open a new issue.

Thank you in advance.

**Comment by rjarry** [comment]
Hi @cristina-defran, thanks for reporting this. I saw the other comment but was too busy to work on the problem.

I will have a look ASAP.
